.github/BOTMETA.yml

```diff
@@ -205,68 +207,33 @@ files:
   $modules/cloud/dimensiondata/dimensiondata_vlan.py:
     authors: tintoy
   $modules/cloud/docker/:
-    authors: chouseknecht
     maintainers: $team_docker
     ignore: cove
     supershipit: felixfontein
   $modules/cloud/docker/docker_compose.py:
-    maintainers: $team_docker sluther
+    maintainers: sluther
     labels: docker_compose
   $modules/cloud/docker/docker_config.py:
     authors: chouseknecht ushuz
   $modules/cloud/docker/docker_container.py:
     authors: ThomasSteinbach chouseknecht cove dusdanig felixfontein kassiansun softzilla zfil
-    maintainers: DBendit WojciechowskiPiotr akshay196 danihodovic dariko jwitko tbouvet
     ignore: ThomasSteinbach cove joshuaconner
-  $modules/cloud/docker/docker_container_info.py:
-    authors: felixfontein
-    maintainers: DBendit WojciechowskiPiotr akshay196 danihodovic dariko jwitko kassiansun tbouvet
-  $modules/cloud/docker/docker_host_info.py:
-    authors: WojciechowskiPiotr
```

Comment:
Shouldn't `authors` entries be kept? (Except for dictories, like the above `authors: chouseknecht`.)

Reply:
the contributor is currently in `team_docker`
why should we keep authors when they are in teams that covers directories?

Reply:
@gundalow , i'm not sure you didn't mean the same yesterday, maybe we can get rid of `authors` completely?
The file is for internal usage by bot, isn't it? To notify people, count their shipits, etc., not for tracking authorship, right?
Is there any difference how the bot handles authors vs maintainers? If no difference, i think, one of them is extra
@gundalow @felixfontein what do you think?

Reply:
There's no difference for the bot, it's just semantics: if someone is later removed from the maintenance team for the group of modules, they are still authors of the module. If we removed them from `authors:` earlier, they are no longer listed there. (If they want to no longer receive notifications, IMO they should be added to `ignore:` and stay as `authors:`.)

Whether or not to keep `authors:` alltogether depends on what happens with the `authors:` in `DOCUMENTATION` validation.

Reply:
`If they want to no longer receive notifications, IMO they should be added to ignore: and stay as authors:`
can't we remove them from a team and put them to `maintainers:` of a certain module (where they are authors) in addition to $team_* instead?
e.g. `maintainers: usernicname $team_postgresql`

Reply:
For people that are no longer involved, you are correct, they should be added to `ignore:`

Do we care about the difference between `author` and `maintainers`?
from a bot notification and merge (ship it) I believe they mean the same
We don't use BOTMETA when generating module docs, so nether maintainer nor author fields will be used to list authors on the webdocs

Reply:
cc @felixfontein

Reply:
It depends on what's the future of `authors:` in `DOCUMENTATION`. That's something we probably have to discuss in the community meeting at some point.

Reply:
Ok https://github.com/ansible/community/issues/539#issuecomment-706947663